kernel/pci/src/lib.rs

```diff
@@ -2,73 +2,70 @@
-pub const PCI_SUBCLASS:              u16 = 0xA;
-pub const PCI_CLASS:                 u16 = 0xB;
-pub const PCI_CACHE_LINE_SIZE:       u16 = 0xC;
-pub const PCI_LATENCY_TIMER:         u16 = 0xD;
-pub const PCI_HEADER_TYPE:           u16 = 0xE;
-pub const PCI_BIST:                  u16 = 0xF;
-pub const PCI_BAR0:                  u16 = 0x10;
-pub const PCI_BAR1:                  u16 = 0x14;
-pub const PCI_BAR2:                  u16 = 0x18;
-pub const PCI_BAR3:                  u16 = 0x1C;
-pub const PCI_BAR4:                  u16 = 0x20;
-pub const PCI_BAR5:                  u16 = 0x24;
-pub const PCI_CARDBUS_CIS:           u16 = 0x28;
-pub const PCI_SUBSYSTEM_VENDOR_ID:   u16 = 0x2C;
-pub const PCI_SUBSYSTEM_ID:          u16 = 0x2E;
-pub const PCI_EXPANSION_ROM_BASE:    u16 = 0x30;
-pub const PCI_CAPABILITIES:          u16 = 0x34;
+pub const PCI_VENDOR_ID:             u8 = 0x0;
+pub const PCI_DEVICE_ID:             u8 = 0x2;
+pub const PCI_COMMAND:               u8 = 0x4;
+pub const PCI_STATUS:                u8 = 0x6;
+pub const PCI_REVISION_ID:           u8 = 0x8;
+pub const PCI_PROG_IF:               u8 = 0x9;
+pub const PCI_SUBCLASS:              u8 = 0xA;
+pub const PCI_CLASS:                 u8 = 0xB;
+pub const PCI_CACHE_LINE_SIZE:       u8 = 0xC;
+pub const PCI_LATENCY_TIMER:         u8 = 0xD;
+pub const PCI_HEADER_TYPE:           u8 = 0xE;
+pub const PCI_BIST:                  u8 = 0xF;
+pub const PCI_BAR0:                  u8 = 0x10;
+pub const PCI_BAR1:                  u8 = 0x14;
+pub const PCI_BAR2:                  u8 = 0x18;
+pub const PCI_BAR3:                  u8 = 0x1C;
+pub const PCI_BAR4:                  u8 = 0x20;
+pub const PCI_BAR5:                  u8 = 0x24;
+pub const PCI_CARDBUS_CIS:           u8 = 0x28;
+pub const PCI_SUBSYSTEM_VENDOR_ID:   u8 = 0x2C;
+pub const PCI_SUBSYSTEM_ID:          u8 = 0x2E;
+pub const PCI_EXPANSION_ROM_BASE:    u8 = 0x30;
+pub const PCI_CAPABILITIES:          u8 = 0x34;
 // 0x35 through 0x3B are reserved
-pub const PCI_INTERRUPT_LINE:        u16 = 0x3C;
-pub const PCI_INTERRUPT_PIN:         u16 = 0x3D;
-pub const PCI_MIN_GRANT:             u16 = 0x3E;
-pub const PCI_MAX_LATENCY:           u16 = 0x3F;
+pub const PCI_INTERRUPT_LINE:        u8 = 0x3C;
+pub const PCI_INTERRUPT_PIN:         u8 = 0x3D;
+pub const PCI_MIN_GRANT:             u8 = 0x3E;
+pub const PCI_MAX_LATENCY:           u8 = 0x3F;
 
 // PCI Capability IDs
-pub const MSI_CAPABILITY:           u16 = 0x05;
-pub const MSIX_CAPABILITY:          u16 = 0x11;
+pub const MSI_CAPABILITY:           u8 = 0x05;
+pub const MSIX_CAPABILITY:          u8 = 0x11;
 
-/// If a BAR's bits [2:1] equal this value, that BAR describes a 64-bit address.
-/// If not, that BAR describes a 32-bit address.
-const BAR_ADDRESS_IS_64_BIT: u32 = 2;
-
-/// The maximum number of PCI buses.
-const MAX_NUM_PCI_BUSES: u16 = 256;
+/// The maximum number of PCI buses, minus 1 (to fit into u8).
+const MAX_NUM_PCI_BUSES_MINUS_1: u8 = 255;
```

Comment:
rename this `MAX_PCI_BUS_NUMBER` to make it clearer

Reply:
I've made it even more clear by saving the range instead